orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/strategy/DetermineSourceServerGroupTask.groovy

```diff
@@ -76,9 +78,6 @@ class DetermineSourceServerGroupTask implements RetryableTask {
       lastException = ex
     }
 
-    Boolean useSourceCapacity = useSourceCapacity(stage, null)
-    boolean isNotFound = (lastException instanceof RetrofitError && lastException.kind == RetrofitError.Kind.HTTP && lastException.response.status == 404)
```

Comment:
fun story, this code only ever worked for about two weeks, as the sourceResolver code was updated to catch the 404 upstream and just return null.